servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1476,6 +1520,183 @@ public void setState(long fileId, SetStateOptions options)
     }
   }
 
+  /**
+   * Schedules a file for async persistence.
```

Comment:
Anyone who wants to schedule an async persistence. Right now checkpoint scheduler calls it, and in a future PR `FileOutStream` will call it when the write type is `ASYNC_THROUGH`.